commit 6dad75c0eee031d91796ef157c68b63745eea6ec
Author: Louwrens van Dellen <lthvd@protonmail.com>
Date:   Wed Jun 26 16:59:54 2019 +0200

    ToFu shouldn't hardcode CC and CXX, they are set by environment.

diff --git a/setup.py b/setup.py
index c50fab3..df6389d 100644
--- a/setup.py
+++ b/setup.py
@@ -59,13 +59,7 @@ except:
     stp = False
 import _updateversion as up
 
-if platform.system() == "Darwin":
-    # make sure you are using Homebrew's compiler
-    os.environ['CC'] = 'gcc-8'
-    os.environ['CXX'] = 'gcc-8'
-else:
-    os.environ['CC'] = 'gcc'
-    os.environ['CXX'] = 'gcc'
+# CC and CXX are set by EasyBuild
 
 not_openmp_installed = check_for_openmp(os.environ['CC'])
 
